Update README.md

PaulDebus opened the pull request:
add more clarification about recompiling the software after adding a custom problem instance

itsoulos commented:
I think that I have added this information in README.md Could you please check it again?

PaulDebus commented:
I just changed some minor wording and added the explicit hint, that the previously installed version if not the correct anymore. Really not a lot. But I think it was more than you had put there.

If you want to merge it, can you fix the conflicts?

itsoulos commented:
Done
